Libraries/LibWeb/Layout/FormattingContext.cpp

```diff
@@ -226,6 +226,20 @@ OwnPtr<FormattingContext> FormattingContext::create_independent_formatting_conte
     }
 }
 
+NonnullOwnPtr<FormattingContext> FormattingContext::create_independent_formatting_context(LayoutState& state, LayoutMode layout_mode, Box const& child_box)
+{
+    auto context = create_independent_formatting_context_if_needed(state, layout_mode, child_box);
+    if (context)
+        return context.release_nonnull();
+
+    auto child_block_container = as_if<BlockContainer>(child_box);
+    if (child_block_container)
+        return make<BlockFormattingContext>(state, layout_mode, *child_block_container, nullptr);
```

Comment:
Nit: We tend to put declarations into the `if` condition where it's readable. eg:

```c++
    if (auto child_block_container = as_if<BlockContainer>(child_box))
        return make<BlockFormattingContext>(state, layout_mode, *child_block_container, nullptr);
```